src/cnd/cnd.ts

```diff
@@ -277,23 +289,53 @@ export class Cnd {
     return info.listen_addresses;
   }
 
+  /**
+   * Sends a swap request to cnd.
+   *
+   * @param swap The details of the swap to initiate.
+   * @returns The URL of the swap request on the cnd REST API.
+   * @throws A {@link Problem} from the cnd REST API or an {@link Error}.
+   */
   public async postSwap(swap: SwapRequest): Promise<string> {
     const response = await this.client.post("swaps/rfc003", swap);
 
     return response.headers.location;
   }
 
+  /**
+   * List the swaps handled by this cnd instance.
+   *
+   * @returns An Array of {@link SwapSubEntity}, which contains details of the swaps.
+   * @throws A {@link Problem} from the cnd REST API or an {@link Error}.
+   */
   public async getSwaps(): Promise<SwapSubEntity[]> {
     const response = await this.fetch("swaps");
     const entity = response.data as Entity;
 
     return entity.entities as SwapSubEntity[];
   }
 
+  /**
+   * Fetch data from the REST API.
+   *
+   * @param path The URL to GET.
+   * @returns The data returned by cnd.
+   * @throws A {@link Problem} from the cnd REST API or an {@link Error}.
+   */
   public fetch<T>(path: string): AxiosPromise<T> {
     return this.client.get(path);
   }
 
+  /**
+   * Proceed with an action request on the cnd REST API.
+   * This request could be a GET HTTP request, which only returns data needed to further proceed with the action,
```

Comment:
This documentation knows more than the function itself (who says we don't do PUT/DELETE requests at some point?). I would discourage from doing that because it makes maintaining the documentation very hard in the long run.

I vaguely remember @da-kami and me actually documenting that function, where did that go?

Reply:
> I vaguely remember @da-kami and me actually documenting that function, where did that go?

I am not sure what you want me to say. You can see from this diff that there were no existing comment.